lib/Cheats.sol

```diff
@@ -1,7 +1,10 @@
 import "forge-std/Test.sol";
 import "./halmos-cheatcodes/src/SymTest.sol";
 
-abstract contract Cheats {
-    Vm internal constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
-    SVM internal constant svm = SVM(address(uint160(uint256(keccak256("svm cheat code")))));
+abstract contract FoundryCheats {
+    Vm internal constant _vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+}
+
+abstract contract HalmosCheats {
```

Comment:
that's almost literally what SymTest is: https://github.com/a16z/halmos-cheatcodes/blob/main/src/SymTest.sol